app/main.py

```diff
@@ -16,4 +16,21 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables = [lucky_number, pi, one_is_a_prime_number,
+             name, my_favourite_films, profile_info,
+             marks, collection_of_coins
+             ]
+immutable_var = (int, float, str, bool, tuple)
+immutable_list = []
+mutable_list = []
+for i in variables:
```

Comment:
Rename i to variable, i is not good variable name

Reply:
ok.

Reply:
done.

Reply:
Hi. When you have a time ,can you close this pull request? thx.
Best Vasyl.